crates/solidity/outputs/cargo/tests/src/cst_output/runner.rs

```diff
@@ -32,22 +31,17 @@ pub fn run(parser_name: &str, test_name: &str) -> Result<()> {
     let mut last_output = None;
 
     for version in VERSION_BREAKS {
-        let version = Version::parse(version)?;
-
         let production_kind = ProductionKind::from_str(parser_name)
             .expect(format!("No such parser: {parser_name}").as_str());
 
-        let output = Language::new(version.to_owned())?.parse(production_kind, &source);
+        let output = Language::new(version.clone())?.parse(production_kind, &source);
 
-        if Some(&output) == last_output.as_ref() {
+        let output = match last_output {
             // Skip this version if it produces the same output.
             // Note: comparing objects cheaply before expensive serialization.
-            continue;
-        }
-
-        last_output = Some(output);
-
-        let output = Language::new(version.to_owned())?.parse(production_kind, &source);
+            Some(ref last) if last == &output => continue,
+            _ => &*last_output.insert(output),
```

Comment:
I wonder why `&*`? is this intended?

Reply:
By default `Option::insert` returns a `&mut T` - while technically it doesn't even matter, I wanted to mark it as immutable for the purposes of the section below, hence why I used a reborrow here. (`let output: &_ = ...` would also do the same thing here)

Reply:
Thank you.